tts-proxy/src/speakers.rs

```diff
@@ -0,0 +1,99 @@
+use std::collections::{HashMap, HashSet};
+
+pub struct TtsSpeakers {
+    pub speakers: HashMap<&'static str, &'static str>,
+    pub speaker_ids: HashSet<&'static str>,
+}
+
+impl TtsSpeakers {
+    /// Returns the speaker id for a given speaker name. If the name is a speaker id, returns a static reference to the string with the same id.
+    pub fn get_speaker_id(&self, speaker_or_id: &str) -> Option<&'static str> {
+        self.speakers
+            .get(speaker_or_id)
+            .or_else(|| self.speaker_ids.get(speaker_or_id))
+            .copied()
+    }
+}
+
+lazy_static::lazy_static! {
```

Comment:
This part might be automated with a build.rs that would extract the speakers from the <select> on vo.codes